src/components/Details/MinutesItemsList/DocumentsList.tsx

```diff
@@ -37,7 +37,7 @@ const DocumentsList: FC<DocumentsListProps> = ({ documents }: DocumentsListProps
     return null;
   }
   return (
-    <details open={isExpanded} onToggle={onToggleIsExpanded}>
+    <details onToggle={onToggleIsExpanded}>
```

Comment:
This just a bugfix? This is only used in the event page correct?

Reply:
Turns out you don't need to control the `open` for `details`. Yea it still works for the event page